leftwm-core/src/handlers/window_handler.rs

```diff
@@ -316,6 +312,7 @@ fn set_relative_floating(window: &mut Window, ws: &Workspace, outer: Xyhw) {
         || ws.center_halfed(),
         |mut requested| {
```

Comment:
Sorry I meant something different in this section. E.g.:
```
requested.center_relative(outer, window.border);
if ws.xyhw.contains_xyhw(&requested) {
    requested
} else {
    requested.center_relative(ws.xyhw, window.border);
}
```

Reply:
center_relative alters the position of requested, so requested would actually need to be the final value in the second clause

Reply:
My code would centre the window on the parent, check whether this new position gets cut off, then if it doesnt returns the position when centred on the parent, if it is it returns the position centred on the workspace

Reply:
I updated the code, is that what you're suggesting?

Reply:
Yeah, does i work as expected? On mobile so cant verify things, but it should work.

Reply:
Well I don't have any issues, so I guess that it works for me. @VuiMuich @M-Mabrouk1 could you please try my latest commit?

Reply:
I am at work right now, but can check later when home.